core-blocks/categories/index.js

```diff
@@ -33,20 +33,11 @@ export const settings = {
 			type: 'boolean',
 			default: false,
 		},
-		align: {
-			type: 'string',
-		},
 	},
 
 	supports: {
 		html: false,
-	},
-
-	getEditWrapperProps( attributes ) {
-		const { align } = attributes;
-		if ( 'left' === align || 'right' === align || 'full' === align ) {
-			return { 'data-align': align };
-		}
+		align: true,
```

Comment:
Align should take the following values:
`align: [ 'left', 'center', 'right', 'full' ],`
to make it compatible with the previous implementation.

Reply:
Excluded categories block as it is a server-side rendered one in [this](https://github.com/WordPress/gutenberg/pull/7100/commits/202105be26cf00eadc93778f5e76030a6d162e11) commit. Thank you ❤️